modules/ROOT/pages/ch2-api.adoc

```diff
@@ -0,0 +1,376 @@
+= CloudHub 2.0 REST API
+
+*Last update* 02-Mar-22
+
+Added all the endpoints from https://github.com/mulesoft/rtf-management-service/blob/master/assets/raml/public/api.raml[GitHub].
+
+* *Private Spaces*
+** Get the list of private spaces in the organization.
+** Create a private space.
+** Delete the private space.
+** Get the private space. *What does this mean?*
+** Update an existing private space.
+
+* *Connections*
+** Get the list of connections inside the private space.
+** Create a private space connection. *What is this?*
+** Delete the private space connection. 
+** Get the private space connection.
+** Update an existing private space connection.
+** Download the connection guide for a VPN connection.
+
+* *VPN*
+** Add a VPN to an existing connection.
+** Delete a VPN from a connection.
+** Update a VPN connection.
+** Get supported VPN configurations.
+
+
+* *Transit Gateway Attachments*
```

Comment:
Transit Gateway actions can be categorized into 2 groups:

Standalone (Transit Gateways)
- List Transit Gateways
- Update a Transit Gateway
- Remove a Transit Gateway

Attachments to Private Spaces (Transit Gateway attachments)
- Create and attach a new Transit Gateway to a Private Space
- List all Transit Gateway attachments in a Private Space
- Get the details of a single Transit Gateway attacment
- Attach an existing Transit Gateway to a Private Space
- Remove a Transit Gateway attachment in a Private Space
- Update routes in a Transit Gateway attachment

I know it can be very confusing. Hope this clears things up.

Reply:
Updated